notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala

```diff
@@ -124,7 +140,11 @@ class DistCopyMapper extends Mapper[NullWritable, Mapping, NullWritable, NullWri
           outputStream => from.withStreamMultipart(
             Bytes(partSize)
             , in => {
-              val counted = DownloadInputStream(in, i => totalBytesDownloaded.increment(i))
+              val counted = DownloadInputStream(in, i => {
+                totalBytesDownloaded.increment(i)
+                totalMegaBytesDownloaded.increment(i / 1024 / 1024)
+                totalGigaBytesDownloaded.increment(i / 1024 / 1024 / 1024)
```

Comment:
> I wanted to point out that doing this way means if i is less then 1MB or 1GB nothing will increment

Yeh the way i read it i would agree that this would happen too, the MB and GB counters would never actually go up unless there was a spike